python/pyfeelpp/feelpp/__init__.py.in

```diff
@@ -175,10 +175,11 @@ def functionSpace( mesh, space="Pch", order=1, worldscomm=None):
     """
     if worldscomm is None:
         worldscomm = Environment.worldsComm(1)
+    worldscomm_list = [worldcomm for worldcomm in worldscomm]
```

Comment:
@prudhomm With the adding of the header to `pybind11/stl.h`, the constructor for `spaces[key]` was no longer waiting a `feelpp._core.WorldsComm` but a `List[feelpp._core.WorldComm]`.
I found this workaround by converting it by hand to a Python list, but there may be a better way to do it...

#2139